### ISSUE TITLE
St. Norbert (ID# 4787)

### ISSUE THREAD

**Issue by StuMackenzie** [MotusDev/Motus-TO-DO#896]
Station to be taken down and all data flagged. Serious aliasing at a PUMA release site, or problem station. I suspect aliasing.

We have to do something with these stations/scenarios.

**Comment by jsayers-bsc** [comment]
One thing I've been noticing is that when a project orders a large quantity of tags they tend to all get issued with the same BI. This strikes me as a bad idea, especially in cases where most of those tags will all be active at the same time in the same area (like this one). Seems it creates the most ideal ideal conditions for aliasing. A better approach might be for Lotek to provide a more varied selection of BI and ID when supplying a project with a lot of tags. If so that's not really in our hands, though maybe worth discussing with Lotek.

**Comment by cjardine-bsc** [comment]
I'm not sure that this really increases the risk of aliasing does it?  In practice the tags are not turned on at the same time, so the chances of their BIs overlapping is fairly low?

**Comment by StuMackenzie** [comment]
Hard to say, I think collisions happen regardless when so many tags are chirping in close proximity.

**Comment by cjardine-bsc** [comment]
FYI I've been doing some testing of filtering using this station. Don't be alarmed if it's data doesn't display as expected right now :-)

**Comment by StuMackenzie** [comment]
Closing in deference to https://github.com/MotusDev/Motus-TO-DO/issues/779 - issue will be discussed and solved there.

**Comment by cjardine-bsc** [comment]
I admire your closing efforts, but this is about filtering based on deployment ID with a single project exception. I'm actively working on this one.

**Comment by jsayers-bsc** [comment]
> I'm not sure that this really increases the risk of aliasing does it? In practice the tags are not turned on at the same time, so the chances of their BIs overlapping is fairly low?

Quite often I think many of them are all one at the same time. Breeding colonies obviously but even just tagging outings will often have many of the deployed tags still present and active during and overlapping time.

In these cases, having multiple tags with the exact same BI transmitting at the same time just invites aliasing issues since any combination of any of those tags could potentially be interpreted as a different tag. All it would take is just a minor corruption of tag signal or how its recorded which happens to match an existing tag. Just about any combination of any tags present could match expected BIs.

And looking at many of the false positive "events" this does seem to be what's happening. Basically all of the PUMA tags were BI 9.7, as were nearly all the false positives (certainly the obvious one). Same with the Nightjar work in Torrance Barrens. They were issued exclusively 4.9 BI tags in 2019 by the looks of it, and no surprise there are ton of false positives also with BI 4.9. (see [here ](https://motus.org/data/receiver/deployment/detections/timeline?id=4777&year=2019)and [here](https://motus.org/data/receiver/deployment/detections/timeline?id=5521&year=2019))

**Comment by cjardine-bsc** [comment]
By "turned on" I don't mean "are on" I mean the moment they are activated initially. 

My (rudimentary) understanding is that having the same burst interval would only compound aliasing if the tags all started bursting at the exact same time. That would mean the bursts would be happening at the exact same time leading to aliasing. 

As it is, having the same BI doesn't seem any better or worst than them having different AIs given they each start their BIing at a different point in time.

**Comment by denislepage** [comment]
Adam, do you know what is the longest run that corresponds to a purported alias tag? And, are the alias tags having a high ratio of missed bursts?

The thing I have never understood about aliasing is that the conditions under which alias tags are created need to be repeatable over multiple intervals to be detected. That implies, in theory, at least 2 tags with the same BI that overlap their burst. For tags with different BI, they’d have to overlap over a much longer pattern that should exceed 20 burst intervals, which tagfinder would normally exclude (that’s the reason BI are based on prime numbers, so no tag BI matches the multiple of a shorter tag).

I still think the way to understand this better with be to run simulated data (with focusing on aliasing only, without radio noise), but the attempts we’ve made so far to push on that have fallen flat.

The few examples that were pointed out to us didn’t align with my theoretical understanding. For instance, there were simultaneous runs created for at least one of the real tags involved as well as the potential alias.

I’m wondering if there is much value in keeping runs that have a substantial number of missed bursts overall, and if this wouldn’t help with lowering the false detections, including aliases.

**Comment by adamdsmith** [comment]
Probably unique situations, but I have several aliased detections of runLen > 30, with 20+ consecutive hits. That's not the norm, obviously. Full disclosure, I've not looked closely at the aliased detections to see the patterns of hit detections...

**Comment by jsayers-bsc** [comment]
> By "turned on" I don't mean "are on" I mean the moment they are activated initially.
> 
> My (rudimentary) understanding is that having the same burst interval would only compound aliasing if the tags all started bursting at the exact same time. That would mean the bursts would be happening at the exact same time leading to aliasing.
> 
> As it is, having the same BI doesn't seem any better or worst than them having different AIs given they each start their BIing at a different point in time.

Right... Yeah I'm also not positive if bursts from multiple tags would be interpreted as coming from different tags unless they happened to be turned on at intervals which corresponded to a real BI internal (or accepted multiple of one). It's certainly more likely with shorter BI. There certainly seems to be something related to BI based on many of the examples I've that involve many tags with the same BI active in the same location, and then the vast majority of the false positives also sharing that same BI
